modules/ipi-install-diagnosing-duplicate-mac-address.adoc

```diff
@@ -0,0 +1,48 @@
+[id="ipi-install-diagnosing-duplicate-mac-address_{context}"]
+= Diagnosing a duplicate MAC address when provisioning a new host in the cluster
+
+[role="_abstract"]
+If the MAC address of an existing bare-metal node in the cluster matches the MAC address of a bare-metal host you are attempting to add to the cluster, the Ironic installation Operator associates the bare metal host with the existing node. If the host enrollment, inspection, cleaning, or other Ironic steps fail, the `metal3-baremetal-operator` continuously retries the install. The Operator displays a registration error for the failed bare metal host. You can diagnose a duplicate MAC address by examining the `metal3-baremetal-operator` log.
```

Comment:
Please use "bare metal" in plain text, at least with host. (If some psycho invented "bare-metal node" to mean a piece of software, that's pathological.)

"Ironic installation Operator" reads like it is the full name of an Operator.  If it is, please treat it with initial caps according to the [doc guidelines](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#operator-name-capitalization).

nitty sugg: does "the `metal3-baremetal-operator` retries the installation continuously" read any smoother to you?

Reply:
Thanks for review Mike :) https://redhat-documentation.github.io/supplementary-style-guide/#bare-metal-adj "bare-metal" with hyphen is correct for adjectival usage, but note also there is a noun usage - confusing.  

I've applied your other suggestion.